packages/design-system/src/components/ChoiceList/Choice.tsx

```diff
@@ -112,13 +116,16 @@ export const Choice = (props: ChoiceProps) => {
 
   let errorId;
   let errorElement;
-  if (props.errorMessage) {
+  if (!props._choiceChild) {
     errorId = props.errorId ?? `${props.id}__error`;
     errorElement = (
       <InlineError id={errorId} inversed={props.inversed} className={props.errorMessageClassName}>
         {props.errorMessage}
       </InlineError>
     );
+  } else {
```

Comment:
This condition needed to be added to strip `_choiceChild` from rendered markup (failed snapshot tests).